mds/apis/prv_api/urls.py

```diff
@@ -5,13 +5,25 @@
 from . import authent, provider_api, providers, service_areas, vehicles
 
 
+class OptionalSlashRouter(routers.DefaultRouter):
+    """Make routes trailing slashes optional.
+
+    `routers.DefaultRouter(trailing_slash=False)` forbids them
+    Inspired by https://stackoverflow.com/a/46163870
+    """
+
+    def __init__(self):
+        self.trailing_slash = '/?'
+        super().__init__()
```

Comment:
Nitpicking but calling `super()` is usually the first thing. Also, why not a keyword argument, or a class attribute?

Reply:
Well catched, this was unncessary. Copy/pasting code...

Reply:
And as a keyword argument to `DefaultRouter`?